### ISSUE TITLE
configuration

### ISSUE THREAD

**Issue by skaji** [perl-actions/install-with-cpm#4]
I think it is reasonable for install-cpm to have some options:
```
uses: perl-actions/install-cpm@v1.0
with:
  perl: /usr/local/bin/perl
  path: /usr/local/bin/cpm
  version: 0.991
  sudo: false
```

Here
* `perl` Which perl to be used with cpm. If this is specified, rewrite the shebang in cpm.
* `path` The install path of cpm. The default value would be `$Config{installsitescript}/cpm`.
* `version` The version of cpm. Note that you can fetch cpm with a specific version by `https://raw.githubusercontent.com/skaji/cpm/<version>/cpm` url. 
* `sudo` Use sudo when locating cpm.

**Comment by atoomic** [comment]
excellent going to add them in addition to the additional I got
```
with:
   install: 'Module'
   cpanfile: 'cpanfile' # one or more modules
   tests: true | false
   global: true | false # control -g
   args: '--any --extra --args for installation'
```

**Comment by skaji** [comment]
Oh, so do you intend this action not only to install cpm itself but also to install modules with cpm?

**Comment by atoomic** [comment]
yes, installing other modules will be optional
I was wondering if instead of version using a `from: https://... ` which default to `https://git.io/cpm` would not be more flexible?

**Comment by atoomic** [comment]
I'm going to implement `perl` but I think it s useless as you can set
```
env: PATH=.....
```

**Comment by atoomic** [comment]
this is going to be fixed in release v1.1
